zetaclient/chains/evm/signer/signer.go

```diff
@@ -39,7 +39,10 @@ import (
 	"github.com/zeta-chain/zetacore/zetaclient/zetacore"
 )
 
-var _ interfaces.ChainSigner = &Signer{}
+var (
+	zeroValue                        = int64(0)
```

Comment:
Seems like to be only used as big.Int, why not directly defining it as big.Int?

Also need comment why the variable is necessary